packages/api/orm/entity/Application.ts

```diff
@@ -3,7 +3,7 @@ import { Environment } from "./Environment";
 import { Portfolio } from "./Portfolio";
 import { ProvisionableEntity } from "./ProvisionableEntity";
 
-@Entity()
+@Entity("application")
```

Comment:
This explicitly define the name of the entity, rather than the default given by TypeORM?